replace default flags with kong.

peter-svensson opened the pull request:
Refactored some things to make it more consistent.
Added verbose flag, not implemented though
Fixes #51

peter-svensson committed:
replace default flags with kong.

Refactored some things to make it more consistent.
Added verbose flag, not implemented though
Fixes #51

argoyle commented on install.sh:
```diff
@@ -81,7 +81,7 @@ tag_to_version() {
     exit 1
   fi
-  # if version starts with 'v', remove it
+  # if args starts with 'v', remove it
```
Unintended change?

```suggestion
  # if version starts with 'v', remove it
```

argoyle commented on pkg/build/build_test.go:
```diff
@@ -13,7 +13,10 @@ import (
 	"testing"
 
 	"github.com/buildtool/build-tools/pkg"
+	"github.com/buildtool/build-tools/pkg/args"
 	"github.com/buildtool/build-tools/pkg/docker"
+	"github.com/buildtool/build-tools/pkg/version"
+
 	"github.com/docker/docker/pkg/archive"
 	"github.com/stretchr/testify/assert"
 )
```
How come local imports are in the middle instead of in the end here? Automatic format?

argoyle commented on www/docs/faq.md:
````diff
@@ -8,7 +8,7 @@ This might cause issues if your docker server is running an older version.
 Errors like:
 ```shell
-Error response from daemon: client version 1.41 is too new. Maximum supported API version is 1.40
+Error response from daemon: client args 1.41 is too new. Maximum supported API args is 1.40
````
Unintended change?
```suggestion
Error response from daemon: client version 1.41 is too new. Maximum supported API version is 1.40
```

argoyle commented on www/docs/installation.md:
````diff
@@ -60,5 +60,5 @@ go build  ./cmd/build/build.go
 
 ```sh
-./build --version
+./build --args
````
Unintended change?
```suggestion
./build --version
```

argoyle reviewed (changes requested):
Sorting of imports seems to be strange in general and a few unintended changes of version to args but otherwise looks good.

peter-svensson committed:
chore: formatting

peter-svensson committed:
docs and small fixes for kubecmd.

Renamed flag `skip-login` to `no-login` to align with `no-pull`
Added doc for `kubecmd`
Updated `kubecmd` with default namespace if missing
Updated docs with short flags

argoyle reviewed (approved):
🚀
